assignment1.txt

```diff
@@ -0,0 +1,10 @@
+1. The return zero is used because it tells the compiler that the code is done. Every time the code runs it checks for errors and the 0 tells the compiler that the program worked if it reached the 0.
+
+
+2. I don't see anything wrong with the code
```

Comment:
Unfortunately, there are a few errors! Look at the code again and comment back with any errors you see! If you still can't find anything wrong, see me and we can go over it together.